trading_calendars/exchange_calendar_xkrx.py

```diff
@@ -521,8 +521,27 @@ class XKRXExchangeCalendar(PrecomputedTradingCalendar):
     Open Time: 9:00 AM, KST (Korean Standard Time)
     Close Time: 3:30 PM, KST (Korean Standard Time)
 
+    NOTE: Korea observes Standard Time year-round.
+
     Due to the complexity around the Korean holidays, we are hardcoding
     a list of holidays covering 1986-2019, inclusive.
+
+    Regularly-Observed Holidays:
+    - Seollal (New Year's Day)
+    - Independence Movement Day
+    - Labor Day
+    - Substitution Holiday
+    - Buddha's Birthday
+    - Memorial Day
+    - Provincial Election Day
+    - Liberation Day
+    - Chuseok (Korean Thanksgiving)
+    - National Foundation Day
+    - Christmas Day
+    - End of Year Holiday
+
+    NOTE: Hangeul Day became a national holiday in 2013
```

Comment:
Could you add a short test asserting that Hangeul Day (October 9th) is a session in 2012 but not a session in 2013?

Reply:
Sure! fwiw I think it should be the other way around; Hangeul Day should _not_ be a session in 2012, and only starts to be a Market-closing Holiday in 2013

Reply:
nvm chatted IRL